// @slidev/parser/dist/index.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});
















var _chunkN3D22V5Pjs = require('./chunk-N3D22V5P.js');

















exports.detectFeatures = _chunkN3D22V5Pjs.detectFeatures; exports.filterDisabled = _chunkN3D22V5Pjs.filterDisabled; exports.getDefaultConfig = _chunkN3D22V5Pjs.getDefaultConfig; exports.mergeFeatureFlags = _chunkN3D22V5Pjs.mergeFeatureFlags; exports.parse = _chunkN3D22V5Pjs.parse; exports.parseAspectRatio = _chunkN3D22V5Pjs.parseAspectRatio; exports.parseRangeString = _chunkN3D22V5Pjs.parseRangeString; exports.parseSlide = _chunkN3D22V5Pjs.parseSlide; exports.prettify = _chunkN3D22V5Pjs.prettify; exports.prettifySlide = _chunkN3D22V5Pjs.prettifySlide; exports.resolveConfig = _chunkN3D22V5Pjs.resolveConfig; exports.resolveFonts = _chunkN3D22V5Pjs.resolveFonts; exports.scanMonacoModules = _chunkN3D22V5Pjs.scanMonacoModules; exports.stringify = _chunkN3D22V5Pjs.stringify; exports.stringifySlide = _chunkN3D22V5Pjs.stringifySlide; exports.verifyConfig = _chunkN3D22V5Pjs.verifyConfig;
